### PR TITLE
Initial actions-based CI

### PR DESCRIPTION
**Pull request by hrboyceiii**
Add initial GH Action workflow for nodejs app

### CODE REVIEW

**Comment by alexturner-incyclesoftware** [comment]
Not bad for a beginner

**Comment by dweissenborn** [comment]
Tacos.

**Review by alexturner-incyclesoftware — COMMENTED** [review]
Looks good to me. We need a solid approach to using PowerShell in actions

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
```
Comment (dweissenborn) [inline .github/workflows/nodejs.yml:16]: I'm commenting multiple lines!

**Review by dweissenborn — APPROVED** [review]

**Review by alexturner-incyclesoftware — APPROVED** [review]

**Pull request merged**